requirements/docs.txt

```diff
@@ -7,3 +7,5 @@ sphinx-copybutton
 pytest-runner
 scikit-learn
 dask[array]>=0.9.0
+# cloudpickle is necessary to provide the 'processes' scheduler for dask
+cloudpickle>=0.2.1
```

Comment:
According to the https://github.com/scikit-image/scikit-image/pull/3582/files, this should be added also to `requirements/optional.txt`.

Reply:
I'm not too sure about that. It is only useful if you want the `'processes'` backend for computation.

See related PR and the issue it closes.
https://github.com/dask/dask/pull/4342

Reply:
Hmmm, I'm surprised that cloudpickle isn't required for the distributed backend?

Reply:
>I'm not too sure about that. It is only useful if you want the 'processes' backend for computation.

I guess it wouldn't harm to add it to avoid potential further issues (i.e. when those 2 backends are chosen by user)?

Reply:
but as dask isn't part of the depenencies anymore, should it be part of the "optional" dependencies?